biz.aQute.repository/src/aQute/maven/provider/POM.java

```diff
@@ -417,13 +417,27 @@ public Archive binaryArchive() {
 
 	@Override
 	public Map<Program, Dependency> getDependencies(MavenScope scope, boolean transitive) throws Exception {
-		return getDependencies(EnumSet.of(scope), transitive);
+		return getDependencies(EnumSet.of(scope), transitive, false);
+	}
+
+	@Override
+	public Map<Program, Dependency> getDependencies(MavenScope scope, boolean transitive,
+		boolean useDependencyManagement) throws Exception {
+		return getDependencies(EnumSet.of(scope), transitive, useDependencyManagement);
+	}
+
+	public Map<Program, Dependency> getDependencies(EnumSet<MavenScope> scope, boolean transitive,
+		boolean useDependencyManagemen) throws Exception {
```

Comment:
Name this `dependencyManagement` to match other places.

Reply:
This one I would keep, becuse we already have a field that represents the `dependencyManagement` of the pom.

Reply:
Same here. This one is also misspelled anyway.